src/main/java/com/palantir/stash/stashbot/managers/PluginUserManager.java

```diff
@@ -56,7 +56,7 @@ public PluginUserManager(UserAdminService uas, PermissionAdminService pas, UserS
     }
 
     public void createStashbotUser(JenkinsServerConfiguration jsc) {
-        StashUser user = us.getUserByName(jsc.getStashUsername());
+        ApplicationUser user = us.getUserByName(jsc.getStashUsername());
```

Comment:
as long as you're s/StashUser/ApplicationUser/ everywhere, may as well do it to your methods?